draft-ietf-rats-architecture.md

```diff
@@ -1188,64 +1188,80 @@ in terms of time(RX)-time(RG), then the Relying Party can check
-
-At time(AA) the Attesting Environment is able to trigger an event
-(e.g. based on an Event-Condition-Action model) to create attestation
-Evidence that is as recent as possible. In essence, at time(AA) the
-Attesting Environment is aware of new values that where generated at
-time(VG) and corresponding Claim values are collected immediately.
-Consecutively, Evidence based on relevant "old" Claims and the just
-collected "new" Claims is generated at time(EG). In essence, the Claims used to generate the Evidence are generated at various time(VG) before time(AA).
-
-In order to create attestation Evidence at
-at time(AA), the Attester requires a fresh (i.e. not expired)
-centrally generated handle that has been distributed to all involved
-entities.
-
-In general, The duration a handle remains fresh depends on
-the content-type of the handle. If it is a (relative or absolute)
-timestamp, clocks synchronized with a shared and trustworthy source of
-time are required. If another value type is used as a handle, the
-reception time of the handle time(HD) provides an epoch (relative time
-of zero) for measuring the duration of validity (similar to a
-heart-beat timeout). From the point of view of a Verifier, validity of
-Evidence is only given if the handle used in Evidence satisfies
-delta(time(HD),time(EG))distribution-interval.
-
-In this usage scenario, time(VG), time(AA), and time(EG) are tightly
-coupled. Also, the absolute point in time at which a handle is received
-by all three entities is assumed to be close to identical.
-
-## Example 4: Nonce-based Background-Check Model Example
+At time(AA), the Attesting Environment is able to trigger an event (e.g. based on an Event-Condition-Action model) to generate Evidence as soon/recent as possible at time (EG).
+This on-change generated Evidence can be based on a mix of relevant "old" Claim values and just recently generated and collected "new" Claim values.
+The delta between time(VG) and time(AA) -- "delta(time(VG),time(AA))" -- can be very large for corresponding "old" Claim values.
+
+In order to be able to generate valid on-change Evidence, the Attester requires a fresh (i.e. not expired) handle that has been distributed to all involved entities.
+In this example, the handle is composed of a signed timestamp. If the Attester does not include a clock, it is possible that the Attester uses an expired handle and it is the Verifier's duty to detect that during the appraisal of Evidence.
+The validity span of a handle can be encoded into the handle itself or determined by an Appraisal Policy.
+From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "delta(time(HD),time(EG)) \< delta(time(HD),time(HD'))".
```

Comment:
There are three different time(HD)'s in the diagram, which one is this talking about?
Same for time(HD').
The sentence seems to imply it's talking about the Verifiers, but if it's just the Verifier then why are there lines to the Attester and Relying Party, with time(HD) values not used in any security check or protocol message.
This doesn't make sense.

Reply:
I assume this is addressed by one of my earlier replies. This is one of the topics that should be discussed later today (20-07-07).